cpp/test/Ice/hash/Client.cpp

```diff
@@ -111,9 +111,9 @@ Client::run(int argc, char** argv)
         "test -f fa:tcp -p 10001 -h hello.zeroc.com",
         "test @ adapt",
         "test @ adapt2",
-        "test:opaque -t 12 -v abc",
-        "test:opaque -t 13 -v abc",
+        "test:opaque -t 12 -v abcd",
         "test:opaque -t 13 -v abcd",
+        "test:opaque -t 13 -v abce",
```

Comment:
I switched these test cases to match the C# test cases.
Especially because _usually_ the number of digits in a BASE64 encoded value should be a multiple of 4.

----

A BASE64 value uses 6 bits `(64 = 2^6)` of information. A byte uses 8 digits of information.
So, for a BASE64 number to cleanly fit into bytes, the number of digits needs to a multiple of 4, so that:
```
4 * (6 digits) = 24 = 3 * (8 bits)
```